students/jarosz_natalia/lesson_03/calculator.py

```diff
@@ -0,0 +1,86 @@
+def adding(add_var_1, add_var_2):
+    res = add_var_1 + add_var_2
+    return res
+
+
+def subtract(add_var_1, add_var_2):
+    res = add_var_1 - add_var_2
+    return res
+
+
+def multiply(add_var_1, add_var_2):
+    res = add_var_1 * add_var_2
+    return res
+
+
+def divide(add_var_1, add_var_2):
+    res = add_var_1 / add_var_2
+    return res
+
+
+def power(add_var_1, add_var_2):
+    res = add_var_1 ** add_var_2
+    return res
+
+
+def help_function():
+    print("HELP")
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def main():
+    print("Welcome to badly organized calculator:")
+    help_function()
+    while True:
+        print("Enter option:")
+        option = input()
+
+        if option == "q":
+            print("GOOD BYE")
+            break
+        elif option == "h" or option == "?":
+            help_function()
+
+        else:
+            if option != 'a' and option != 's' and option != 'm' and option\
+                    != 'd' and option != 'p':
+                print("Wrong choice, choose another option.")
+                continue
```

Comment:
Instead of checking twice the value of ```option``` better is add ```else``` in line 81 where you just prin message and continue loop :) This is just tip :)